File: 26-remove-duplicates-from-sorted-array/26-remove-duplicates-from-sorted-array.java
class Solution {
    public int removeDuplicates(int[] nums) {
        int idx = 0;
        
        for(int i = 1; i < nums.length; i++)
        {
            if(nums[i] != nums[idx])
            {
                nums[++idx] = nums[i];
            }
        }
        return idx+1;
    }

}

File: 26-remove-duplicates-from-sorted-array/26-remove-duplicates-from-sorted-array.cpp
class Solution
{
    public:
        int removeDuplicates(vector<int> &nums)
        {
            int i = 0, j = 1, n = nums.size(), cnt = 0;
            vector<int> v;
            v.push_back(nums[0]);
            while (j < n)
            {
                if (nums[i] == nums[j]) j++;
                else if (nums[i] != nums[j])
                {
                    cnt++,
                    i = j;
                    v.push_back(nums[i]);
                    j++;
                }
            }
            nums = v;
            return cnt + 1;
        }
};